type-definitions.js

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
 
 export type Component = {
     displayName: string,
     description: string,
     demos: { title: string, render: () => React.Element<*> }[],
```

Comment:
Not needed here anymore, I'll create a separate PR for passing navigation to render.